src/data/accessibility_de.json

```diff
@@ -0,0 +1,55 @@
+            "Wir arbeiten an der baldmöglichsten Beseitigung der aufgelisteten Barrieren.",
+            "<b>Datum der Erstellung bzw. der letzten Aktualisierung der Erklärung</b>",
+            "Diese Erklärung wurde am 21.02.2022 erstellt."
+        ]
+    },
+    "section-report-barriers": {
+        "headline": {
+        "title": "Barrieren melden: Kontakt zu den Ansprechpartnern",
+        "text": "Sie möchten uns bestehende Barrieren mitteilen oder Informationen zur Umsetzung der Barrierefreiheit erfragen? Gerne können Sie sich per <a href='coronawarnapp...rki...de' class='email'>E-Mail</a> oder über unser <a href='https://www.rki.de/SharedDocs/Kontaktformulare/weitere/Corona-Warn-App/Corona-Warn-App_Integrator.html' target='_blank' rel='noopener noreferrer'>Kontaktformular</a> bei uns melden."
+        }
+    },
+    "section-arbitrary-service": {
+        "title": "Schlichtungsverfahren",
+        "textblock": [
+            "Wenn auch nach Ihrem Feedback an den oben genannten Kontakt keine zufriedenstellende Lösung gefunden wurde, können Sie sich an die Schlichtungsstelle nach § 16 BGG wenden. Die Schlichtungsstelle BGG hat die Aufgabe, bei Konflikten zum Thema Barrierefreiheit zwischen Menschen mit Behinderungen und öffentlichen Stellen des Bundes eine außergerichtliche Streitbeilegung zu unterstützen. Das Schlichtungsverfahren ist kostenlos. Es muss kein Rechtsbeistand eingeschaltet werden. Weitere Informationen zum Schlichtungsverfahren und den Möglichkeiten der Antragstellung finden sie auf der <a href='https://www.schlichtungsstelle-bgg.de' target='_blank' rel='noopener noreferrer'>Webseite der Schlichtungsstelle</a>.",
+            "Sie erreichen die Schlichtungsstelle unter folgender Adresse:",
+            "Schlichtungsstelle nach dem Behindertengleichstellungsgesetz bei dem Beauftragten der Bundesregierung für die Belange von Menschen mit Behinderungen<br/>Mauerstraße 53<br/>10117 Berlin<br/>Telefon: +49 (0)30 18 527-2805<br/>Fax: +49 (0)30 18 527-2901<br/>E-Mail: <a href='info...schlichtungsstelle-bgg...de' class='email'>info@schlichtungsstelle-bgg.de<a/><br/>Internet: <a href='https://www.schlichtungsstelle-bgg.de' target='_blank' rel='noopener noreferrer'>https://www.schlichtungsstelle-bgg.de</a>"
```

Comment:
See previous comment on telephone number formatting. Suggest to remove "(0)".

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2508/commits/8b0a4178807bc2a0c8f46aafdf4f47e25b2440fb, thanks!